### PR TITLE
fixed funnel position

### PR DESCRIPTION
**Pull request by Geocali**
At creation of the funnel, the position was specified outside of the component and was not taken into account, so the position was always (0, 0).

This pull request fixes this, adds a test, and some explanations to start contributing to the library.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+pip3 install -r requirements_dev.txt
+```
+
+Run Nifi and Nifi registry on localhost:8080
+```bash
+docker-compose -f resources/docker/latest/docker-compose.yml up
+```
+
+Run the tests
+```bash
```
Comment (ottobackwards) [inline CONTRIBUTING.md:23]: I think you want to run TOX here

Comment (ottobackwards) [reply CONTRIBUTING.md:23]: Also, this project has contributing here : https://github.com/Chaffelson/nipyapi/blob/master/docs/contributing.rst
I'm not sure @Chaffelson would want both.

I also updated contributing in #223

Comment (Geocali) [reply CONTRIBUTING.md:23]: ok, I remove my file then

**Comment by Chaffelson** [comment]
Thanks for this contribution!

**Pull request merged**